data/lessons.json.unvalidated.in

```diff
@@ -264,67 +264,50 @@
                     "type": "regex",
-                    "value": "^always-show-log-out$"
+                    "value": "^gtk-theme$"
                 }],
                 "example": {
-                    "success": "gsettings list-keys org.gnome.shell",
+                    "success": "gsettings list-keys org.gnome.desktop.interface",
                     "failure": "gsettings"
                 }
             },
-            "disable_social_bar": {
+            "get_clock_format": {
                 "input": "console",
-                "mapper": [{
-                    "type": "shell",
-                    "value": {
-                        "hook": "parse_gsettings",
-                        "hook_settings": {
-                            "variant_type": "b"
-                        }
-                    }
-                }, "raw_output", {
-                    "type": "shell_custom",
-                    "value": {
-                        "command": "gsettings get org.gnome.shell enable-social-bar",
-                        "hook": "parse_gsettings",
-                        "hook_settings": {
-                            "variant_type": "b"
-                        }
-                    }
-                }, {
+                "mapper": ["shell", "raw_output", {
                     "type": "regex",
-                    "value": "false"
+                    "value": "^('24h'|'12h')$"
                 }],
                 "example": {
-                    "success": "gsettings set org.gnome.shell enable-social-bar false",
-                    "failure": "gsettings set org.gnome.shell enable-social-bar true"
+                    "success": "gsettings get org.gnome.desktop.interface clock-format",
+                    "failure": "gsettings get org.gnome.desktop.interface clock"
                 }
             },
-            "enable_social_bar": {
+            "change_clock_format": {
                 "input": "console",
                 "mapper": [{
                     "type": "shell",
                     "value": {
                         "hook": "parse_gsettings",
                         "hook_settings": {
-                            "variant_type": "b"
+                            "variant_type": "s"
                         }
                     }
                 }, "raw_output", {
                     "type": "shell_custom",
                     "value": {
-                        "command": "gsettings get org.gnome.shell enable-social-bar",
+                        "command": "gsettings get org.gnome.desktop.interface clock-format",
                         "hook": "parse_gsettings",
                         "hook_settings": {
-                            "variant_type": "b"
+                            "variant_type": "s"
                         }
                     }
                 }, {
                     "type": "regex",
-                    "value": "true"
+                    "value": "^(\"24h\"|\"12h\")$"
```

Comment:
Any reason why above you use the same regex with single quotes instead? Make them consistent if possible.